packages/client/src/AdditionalApi.ts

```diff
@@ -0,0 +1,18 @@
+import { ClientResponse, RepositoryClient } from "./RepositoryClient.js";
+import { LionwebResponse } from "./responses.js"
+
+export class AdditionalApi {
+    client: RepositoryClient
+    
+    constructor(client: RepositoryClient) {
+        this.client = client
+    }
+
+    async getNodeTree(nodeIds: string[]): Promise<ClientResponse<LionwebResponse>> {
+        console.log(`AdditionalApi.getNodeTree`)
+        const x = await this.client.postWithTimeout(`additional/getNodeTree`, { body: { ids: nodeIds }, params: "" })
```

Comment:
Is this variable useful? I would probably inline it in the return or renaming it

Reply:
Removed variable, has no use at all.